milton-processor/scribe.ts

```diff
@@ -27,6 +27,15 @@ app.get("/cached", async (req, res) => {
   res.status(200).send(articleCached);
 });
 
+app.get("/fetch", async (req, res) => {
+  res.set('Access-Control-Allow-Origin', CONFIG.cors_client);
+
+  const url = req.query.url.toString();
```

Comment:
Out of curiosity, why is `toString` needed here?

Reply:
To convert the URL into a string obviously...

![](https://i.giphy.com/media/cEYFeDKVPTmRgIG9fmo/giphy.gif)